deploy/osps/default/osp-centos.yaml

```diff
@@ -104,7 +104,7 @@ spec:
 
           yum install -y \
               docker-ce-cli-19.03* \
-              containerd.io-1.4* \
+              containerd.io-1.5* \
               docker-ce-19.03* \
```

Comment:
We haven't tested this combination in machine-controller; there, we pinned the containerd version installed with docker to 1.4.

Reply:
But it seems docker is installed unconditionally in any profile? That's also not matching machine-controller behaviour right now. We might need to sync them on this.

Reply:
Is there any significance behind just updating this for `1.24`?